lib/pleaserun/platform/base.rb

```diff
@@ -130,6 +130,13 @@ class InvalidTemplate < ::StandardError; end
     validate { |v| v == "ulimited" || v.to_i > 0 }
   end
 
+  attribute :envs, "Env key/value pairs to insert into init script (Upstart only)" do
```

Comment:
I am in favor of this, but would prefer this attribute be called 'environment_variables'. Also, this attribute can be put inside the upstart platform class instead of base.